features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/DefaultImageWireframeHelper.kt

```diff
@@ -24,16 +24,15 @@ import com.datadog.android.sessionreplay.utils.ImageWireframeHelper
 import com.datadog.android.sessionreplay.utils.ViewIdentifierResolver
 import java.util.Locale
 
-// This should not have a callback but it should just create a placeholder for base64Serializer
-// The resourcesSerializer dependency should be removed from here
-// TODO: RUM-0000 Remove the resourcesSerializer dependency from here
+// This should not have a callback but it should just create a placeholder for resourceResolver
+// The resourceResolver dependency should be removed from here
+// TODO: RUM-0000 Remove the resourceResolver dependency from here
 internal class DefaultImageWireframeHelper(
     private val logger: InternalLogger,
-    private val resourcesSerializer: ResourcesSerializer,
-    private val imageCompression: ImageCompression,
+    private val resourceResolver: ResourceResolver,
     private val viewIdentifierResolver: ViewIdentifierResolver,
-    private val viewUtilsInternal: ViewUtilsInternal,
-    private val imageTypeResolver: ImageTypeResolver
+    private val viewUtilsInternal: ViewUtilsInternal = ViewUtilsInternal(),
+    private val imageTypeResolver: ImageTypeResolver = ImageTypeResolver()
```

Comment:
what is the point of default values if both constructor calls in prod and test files are providing these properties explicitly?

Reply:
Looks like a mistake on the rebase - I'll remove it